docs/adr/0002-use-cc0-1-0-as-license.md

```diff
@@ -0,0 +1,32 @@
+# Use CC0-1.0 as license
```

Comment:
Is the aim of this ADR to use this licence ? Then we would not have considered other licences.
I suggest `Choose and apply a licence to the repository`.

Reply:
According to the [MADR template](https://github.com/adr/madr/blob/master/docs/adr/template.md), the title should be a `[short title of solved problem and solution]`. So the title is right within this aim.

Then the explanations for the choice should be in the ADR itself, which is the case here.

Let me know if that answers your question.

Reply:
OK, could be merged then